contracts/LockedNORI.sol

```diff
@@ -397,35 +446,21 @@ contract LockedNORI is
         grant.lockupSchedule.cliffs[1].amount,
         grant.claimedAmount,
         grant.originalAmount,
-        grant.lastRevocationTime
+        grant.lastRevocationTime,
+        grant.lastQuantityRevoked
       );
   }
 
-  // todo document expected initialzation state
-  function initialize(BridgedPolygonNORI bridgedPolygonNoriAddress)
-    public
-    initializer
-  {
-    address[] memory operators = new address[](1);
-    operators[0] = _msgSender();
-    __Context_init_unchained();
-    __ERC165_init_unchained();
-    __AccessControl_init_unchained();
-    __AccessControlEnumerable_init_unchained();
-    __Pausable_init_unchained();
-    __ERC777PresetPausablePermissioned_init_unchained();
-    __ERC777_init_unchained("Locked BridgedPolygonNORI", "lNORI", operators);
-    _bridgedPolygonNori = bridgedPolygonNoriAddress;
-    _ERC1820_REGISTRY.setInterfaceImplementer(
-      address(this),
-      ERC777_TOKENS_RECIPIENT_HASH,
-      address(this)
-    );
-    _grantRole(TOKEN_GRANTER_ROLE, _msgSender());
+  /**
+   * @notice Unlocked balance less any claimed amount at current block timestamp.
+   */
+  function unlockedBalanceOf(address account) public view returns (uint256) {
```

Comment:
also this one?

Reply:
Sure. Moved them to fix linting errors. Will open a subsequent with these changes